Bio/SeqUtils/IsoelectricPoint.py

```diff
@@ -80,6 +87,7 @@ def __init__(self, protein_sequence, aa_content=None):
         self.sequence = str(protein_sequence).upper()
         if not aa_content:
             from Bio.SeqUtils.ProtParam import ProteinAnalysis as _PA
+
```

Comment:
Is this blank line added by black? If not, I'd prefer to omit it.

Reply:
Black puts a blank line after any import - fine for top level imports, but a bit annoying for lazy imports.

Reply:
Yes added by black.